boule/tests/test_ellipsoid.py

```diff
@@ -202,6 +202,20 @@ def test_normal_gravity_arrays(ellipsoid):
     npt.assert_allclose(gammas, ellipsoid.normal_gravity(latitudes, heights), rtol=rtol)
 
 
+@pytest.mark.parametrize("ellipsoid", ELLIPSOIDS, ids=ELLIPSOID_NAMES)
+def test_normal_gravity_siunits_arrays(ellipsoid):
+    "Compare normal gravity passing arrays as arguments instead of floats and returning values in SI units"
+    rtol = 1e-10
+    heights = np.zeros(3)
+    latitudes = np.array([-90, 90, 0])
+    gammas = np.array(
+        [ellipsoid.gravity_pole, ellipsoid.gravity_pole, ellipsoid.gravity_equator]
+    )
+    npt.assert_allclose(
+        gammas, ellipsoid.normal_gravity(latitudes, heights, si_units=True), rtol=rtol
+    )
+
+
```

Comment:
This could be replaced by another `parametrize` in line 190:

```python
@pytest.mark.parametrize("si_units", [False, True], ids=["mGal", "SI"])
@pytest.mark.parametrize("ellipsoid", ELLIPSOIDS, ids=ELLIPSOID_NAMES)
def test_normal_gravity_arrays(ellipsoid, si_units):
    "Compare normal gravity passing arrays as arguments instead of floats"
    ...
```

You'd also need an `if` to check if you need to convert the `gammas` list.